### ISSUE TITLE
0.1.0 release and publication

### ISSUE THREAD

**Issue by calebrob6** [microsoft/torchgeo#55]
The following list enumerates all tasks that need to be completed before:

* making the repo public,
* tagging a 0.1.0 release,
* publishing on PyPI/conda-forge/Spack, and
* publishing a paper on TorchGeo.

This list may change over time as we reevaluate the remaining tasks that need to be done.

### Documentation

- [x] Decide on a logo
- [x] Add tutorial (#38, #93)
- [x] Add installation instructions (#153)
- [x] Add contributing instructions
- [x] Document release procedure (https://github.com/microsoft/torchgeo/wiki/Releasing-Instructions)

### Continuous Integration

- [x] Set up codecov, add badge (must be public first?)
- [x] Set up RTD, add badge (must be public first?)
- [x] Determine minimum supported dependency versions (#32, #80)
- [x] Achieve 100% test coverage

### Publication

- [x] Decide on a publication venue (preprint on arXiv, paper in ICLR)
- [x] Benchmark different ways of performing the reprojection, etc. (#81)
- [x] Compare no-pretrain/imagenet-pretrain/torchgeo-pretrain
- [x] FInd publications that discuss and compare various geospatial datasets
- [x] Create an Overleaf document with ICLR template

### Datasets

- [x] Implement a few more GeoDatasets
  - [x] Chesapeake (#18)
  - [x] NAIP (#57)
  - [x] MSFT building footprints (#69)
- [x] Refactor indexing and getitem logic from datasets to GeoDataset base class (#73)

### Models

- [x] Set up infrastructure for hosting pre-trained model weights

### Samplers

- [x] Implement GridGeoSampler (#78)
- [x] More intelligent sampling (#82, #84)

### Trainers

- [x] Factor out task specific configuration logic into the different task classes
- [x] Pass the "task level" arguments to the DataModules as well as the Tasks. Batch size/seed/num workers should probably go in the "task level" arguments.
- [x] Add trainer for GeoDataset
- [x] Refactor trainers to reduce code duplication (#205)

### Transforms

- [x] Experiment with different ways of doing augmentation pipelines
- [x] Use Kornia, provide wrapper
- [x] Add geospatial-specific transforms (#127)
